### ISSUE TITLE
Execute C# code in pugzor template

### ISSUE THREAD

**Issue by drewlsvern** [AspNetMonsters/pugzor#21]
Is there currently anyway to execute c# code in the template similar to how you do it in Razor?

Razor Example
<div>@Datetime.Now</div>

**Comment by dpaquette** [comment]
Not C# but you can execute Javascript in the view. 

https://pugjs.org/language/code.html

**Comment by mavanmanen** [comment]
Very doable, "@" has no special meaning in Pug, this piping the output from the Pug compiler into the Razor view should allow for the csharp code to be executed

**Comment by stimms** [comment]
My feeling on this is that as soon as we start allowing C# to be run in the templates we're creating a dialect of pug which cannot be used outside of our system. That seems antithetical to the goals of interoperability.

**Comment by mavanmanen** [comment]
True, that would make it a hybrid Pug/Razor view and not a pure Pug view. But, should that be completely disallowed or the choice given to the user?
